### DISCUSSION TITLE
Why do you remove the right side preview panel and the sroll bar on it?

### DISCUSSION THREAD

**Discussion by Rereyuop** [d2phap/ImageGlass#812] [General] [answered]
### System information:
 - Windows OS version: 2004
 - ImageGlass version: I used moon 7.0.8.5 version before. Now there is no right side thumbnail list exist in latest release. Nethier in old versions I guess. I tried some of them then I noticed right side preview layout is just a test, nerver mind.

### Questions
The point is the scroll bar attach to preview panel, please add it again.
![Snipaste_2020-07-03_02-53-17](https://user-images.githubusercontent.com/56964252/86398804-c4dd0b80-bcd8-11ea-8d48-41d0242a091f.png)

**Comment by fire-eggs** [comment, marked as answer]
There is a setting to view the thumbnail scroll bar. Goto Settings, Image tab:
![image](https://user-images.githubusercontent.com/9809727/86414594-5c6b3b80-bc92-11ea-93be-5886bf0bc215.png)

The clip also shows the option to have the thumbnails on the right side if that is your preference.

**Comment by Rereyuop** [comment]
> There is a setting to view the thumbnail scroll bar. Goto Settings, Image tab:
> ![image](https://user-images.githubusercontent.com/9809727/86414594-5c6b3b80-bc92-11ea-93be-5886bf0bc215.png)
> 
> The clip also shows the option to have the thumbnails on the right side if that is your preference.

Sorry, I didn't look into it carefully. Thank you for your reply. 
By the way, the gif control feature is so good, but how do I make it stop playing?

**Comment by d2phap** [comment]
> how do I make it stop playing?

You can press Space, or click on main menu > Image > Start / pause gif animation
